metroae

```diff
@@ -1644,8 +1645,22 @@ function download_container_templates {
         write_to_screen_and_script_log ""
         exit 1
     fi
-    curl -O $TEMPLATE_TAR_LOCATION
-    curl -O $VSD_SPECIFICATIONS_LOCATION
+    {
+        curl -O $TEMPLATE_TAR_LOCATION &> /dev/null &&
+        curl -O $VSD_SPECIFICATIONS_LOCATION &> /dev/null
+    } || {
+        echo "Warning: Internet connection not detected."
+        echo "Template or VSD Specification files were not downloaded."
```

Comment:
Instead of `Template or VSD Specification files were not downloaded.  ` How about `MetroAE Config Templates and VSD Specification files were not downloaded` ?